### DISCUSSION TITLE
Zeiterfassung nach Projektende für Zeiten vor Projektende

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3236] [Questions & Answers]
Ich habe die Einstellung gefunden das es Benutzern erlaubt ist das Budget zu über buchen und das ich einen Zeitraum konfigurieren kann für eine Übergangszeit nach Monatsende wie lange Zeiteinträge gändert werden können. Leider ist es mir noch nicht gelungen das Benutzer auf ein Projekt was z.B. am 31.03.2022 endet am 01.04.2022 noch Zeiteinträge z.B. für den 31.03. o.ä. nachbuchen. Ist es möglich auch hier eine Übergangfrist zu konfigurieren ?

**Comment by kevinpapst** [comment]
Hallo, wieso sollte das nicht gehen? Wenn das Projekt zeitlich heute geendet ist, erscheint es gar nicht mehr in der Liste der Projekte.
Aber wenn man dann das Datum zurücksetzt und den Kunden auswählt, muss das beendete Projekt in der Projektliste erscheinen.
Das funktioniert aber nur, wenn man die Projektliste nach Änderung des Datums zum Reload zwingt, z.B. durch Auswahl des Kunden.

**Comment by jens-2305** [comment]
Ah ok, das hatte ich noch nicht versucht, das mit dem Reload bei Auswahl des Kunden fehlte mir. Danke für den Hinweis.

**Reply by kevinpapst** [reply]
Ja, ist nicht ideal gelöst, aber wenn man es einmal weiß ... 😄

**Comment by jens-2305** [comment]
Ist es eventuell relativ einfach möglich auch bei Änderung des Datums die Projektliste neu aufzubauen ?

**Reply by kevinpapst** [reply]
"eventuell" und "relativ" sind neben den Klassikern "müsste" und "einfach" sehr beliebte Worte in der Softwareentwicklung^^

**Comment by jens-2305** [comment]
Ich weiß deswegen wars als vorsichtige Frage formuliert ;-) 
Ich fange jetzt auch nicht mit ich "dachte" an

**Reply by kevinpapst** [reply]
Die meisten Features die einfach waren, sind bereits umgesetzt^^ 

Ein Entwickler könnte es eventuell mit dem CustomContent Plugin hinbekommen, da kann man ja eigenes Javascript injecten.

Das Feature ist eigentlich auch nur eine Krücke um menschliche Schwächen auszugleichen. Man sollte regelmäßig und zeitnah seine Zeiten erfassen. Diese Zeiten noch Tage nach Projektende verbuchen zu müssen, sollte hoffentlich selten vorkommen.
